frontend/controller/actions/chatroom.js

```diff
@@ -227,20 +228,25 @@ export default (sbp('sbp/selectors/register', {
         console.warn(`Unable to share rotated keys for ${originatingContractID} with ${pContractID}: Missing CEK`)
         return
       }
-      return {
-        contractID,
-        foreignContractID: pContractID,
-        // $FlowFixMe
-        keys: Object.values(newKeys).map(([, newKey, newId]: [any, Key, string]) => ({
-          id: newId,
-          meta: {
-            private: {
-              content: encryptedOutgoingData(pContractID, CEKid, serializeKey(newKey, true))
-            }
-          }
-        }))
-      }
-    }))
+      return [
```

Comment:
Since above this returns `undefined`... should the resulting `.map` result `.filter` out these `undefined`s?

Reply:
That's got to do with `flow`. I checked now and it seems to be fine to just return, but sometimes it complains if `.map` doesn't return a value.